### PR TITLE
additional solana-sys-tuner limits

### PR DESCRIPTION
**Pull request by jbiseda**
#### Problem
Add additional OS network tuning for burst traffic

#### Summary of Changes
Add `solana-sys-tuner` updated limits for:
net.core.optmem_max=4194304
net.core.netdev_max_backlog=250000

per: https://community.mellanox.com/s/article/linux-sysctl-tuning

Fixes #

### CODE REVIEW

**Comment by mvines** [comment]
If these values stick, please add them to the manual tuning section in the docs as well: https://docs.solana.com/running-validator/validator-start#manual

File: sys-tuner/src/main.rs [hunk 1]
```diff
@@ -93,6 +93,10 @@ fn tune_kernel_udp_buffers_and_vmmap() {
 
     // increase mmap counts for many append_vecs
     sysctl_write("vm.max_map_count", "1000000");
+
+    // Reference: https://community.mellanox.com/s/article/linux-sysctl-tuning
```
Comment (t-nelson) [inline sys-tuner/src/main.rs:97]: This document is referencing TCP tuning, so I poked around for the same vars WRT UDP and found https://indico.cern.ch/event/212228/contributions/1507212/attachments/333941/466017/10GE_network_tests_with_UDP.pdf.  Admittedly it's tuning 10Gbit and using larger MTU, but at least the right protocol.  The suggestions are quite a bit different and also tough a couple other knobs that we haven't investigated.

Additionally, it got me wondering whether we might be pinning PoH to the NIC's IRQ affinity core, which would probably be bad.

WDYT about rolling new values out to our nodes for a week and monitoring before we change the code?

Comment (jbiseda) [reply sys-tuner/src/main.rs:97]: Yeah, let's try on our nodes first.

**Comment by McSim85** [comment]
Hey @jbiseda @t-nelson 
How was your testing going?

I am asking if we should change the zero value to something worthable. 
In practice, setting `net.core.optmem_max` to infinite value could potentially cause problems, since it would allow a single socket to allocate a huge amount of memory for options.

**Comment by jbiseda** [comment]
> Hey @jbiseda @t-nelson How was your testing going?

sys-tuner has been removed, see: #31682

**Pull request closed without merging**